userspace/libscap/engine/gvisor/fillers.h

```diff
@@ -349,7 +349,7 @@ fill_event_dup3_x(scap_sized_buffer scap_buf, size_t* event_size, char* scap_err
                   int64_t res,
                   int64_t oldfd,
                   int64_t newfd,
-                  uint32_t flags);
+                  int32_t flags);
```

Comment:
Similarly to the kernel drivers this resolves to the scap flags (see below) and therefore can be reverted.

```
ret.status = scap_gvisor::fillers::fill_event_dup3_x(
			                 scap_buf, &ret.size, scap_err,
			                 gvisor_evt.exit().result(),
			                 gvisor_evt.old_fd(),
			                 gvisor_evt.new_fd(),
			                 dup3_flags_to_scap(gvisor_evt.flags()));
```

@LucaGuerra would you mind helping check on `gvisor_evt.flags()` ty!